library/include/hipblas.h

```diff
@@ -19864,6 +19901,45 @@ HIPBLAS_EXPORT hipblasStatus_t hipblasScalEx(hipblasHandle_t   handle,
                                              int               incx,
                                              hipblasDatatype_t executionType);
 
+/*! \brief BLAS EX API
+
+    \details
+    hipblasScalEx_v2 acts identically to hipblasScalEx but accepts hipDataType
+    rather than hipblasDatatype_t for alphaType, xType, and executionType.
+
+    @param[in]
+    handle    [hipblasHandle_t]
+              handle to the hipblas library context queue.
+    @param[in]
+    n         [int]
+              the number of elements in x.
+    @param[in]
+    alpha     device pointer or host pointer for the scalar alpha.
+    @param[in]
+    alphaType [hipDataType]
+               specifies the datatype of alpha.
+    @param[inout]
+    x         device pointer storing vector x.
+    @param[in]
+    xType [hipDataType]
+           specifies the datatype of vector x.
+    @param[in]
+    incx      [int]
+              specifies the increment for the elements of x.
+    @param[in]
+    executionType [hipDataType]
+                   specifies the datatype of computation.
+
+    ********************************************************************/
+HIPBLAS_EXPORT hipblasStatus_t hipblasScalEx_v2(hipblasHandle_t handle,
+                                                int             n,
```

Comment:
I do not think we need to document the `hipblasScalEx_v2` function. We hope the user compiles with `-D HIPBLAS_V2` and calls hipblasScalEx and hipblasScalEx is converted to hipblasScalEx_v2 by the cpp preprocessor. 
Another way it could be documented is like https://github.com/ROCmSoftwarePlatform/rocBLAS/blob/release/rocm-rel-5.6/library/include/internal/rocblas-functions.h#L12763